frontend/ui/quickstart.lua

```diff
@@ -10,61 +10,185 @@ local lfs = require("libs/libkoreader-lfs")
 local _ = require("gettext")
 
 local QuickStart = {
-    quickstart_force_show_version = 201511982,
+    quickstart_force_show_version = 2021070000,
 }
 
 local language = G_reader_settings:readSetting("language") or "en"
 local version = Version:getNormalizedCurrentVersion()
 local rev = Version:getCurrentRevision()
 
+local stylesheet = [[
+div.logo { float: right; }
+div.logo > img { height: 4em; }
+h1 { page-break-before: avoid; margin-bottom: 1em; }
+h2 { background-color: black; color: white; text-align: center; page-break-after: avoid; }
+
+hr { margin: 1em 20%; }
+div.generated { font-size: x-small; }
+
+li { margin: 0.5em 0; page-break-inside: avoid; }
+blockquote { text-align: center; page-break-before: avoid; page-break-inside: avoid; } /* Markdown lines starting with '>' */
+
+/* Inline image (icons) */
+p img, blockquote img { height: 1.5em; vertical-align: bottom; }
+
+/* Standalone image (UI screenshots) */
+div.img-block { text-align: center; margin: 0.5em; }
+div.img-block > img { max-height: 60vh; max-width: 80vw; }
+div.break-before-avoid { page-break-before: avoid; }
+div.break-after-avoid { page-break-after: avoid; }
+
+div.table { display: table; page-break-inside: avoid; width: 100%; }
+div.table > div { display: table-row; }
+div.table > div > * { display: table-cell; text-indent: 0; padding: 0.3em; }
+div.table > div > *:nth-child(2) { text-align: left; hyphens: none; background-color: #eeeeee; }
+div.table > div > *:nth-child(3) { white-space: nowrap; }
+]]
+
 local quickstart_guide = T(_([[
-# KOReader Quickstart Guide
+<div class="logo">![KOReader](../resources/koreader.svg)</div>
+
+# QUICKSTART GUIDE
+
+* [USER INTERFACE](#ui)
+* [USER INTERFACE TIPS](#uitips)
+* [ACCESSING FILES](#afiles)
+* [TRANSFERRING FILES](#tfiles)
+* [FRONTLIGHT/BACKLIGHT](#flight)
+* [WHILE READING](#reading)
+* [INSTALLING DICTIONARIES](#dicts)
+* [MORE INFO](#more)
+
+---
+
+You can access the complete user manual from [our GitHub page](https://github.com/koreader/koreader).
+
+## USER INTERFACE <a id="ui"></a>
+
+<div class="img-block">![Touch zones](../resources/quickstart/touchzones.png)</div>
+
+- To show the **TOP MENU** or **BOTTOM MENU** you can click the indicated zones. You can click or swipe down the upper zone to show the **TOP MENU**.
+
+- **STATUS BAR** zone can be used to cycle between STATUS BAR items if one item is visible. This will also hide and show the STATUS BAR if you tap enough times.
```

Comment:
```suggestion
- The **STATUS BAR** zone can be used to cycle between STATUS BAR items if one item is visible. This will also hide and show the STATUS BAR if you tap enough times.
```